sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/ObjectHashAggregateExec.scala

```diff
@@ -74,12 +74,17 @@ case class ObjectHashAggregateExec(
 
   override lazy val metrics = Map(
     "numOutputRows" -> SQLMetrics.createMetric(sparkContext, "number of output rows"),
-    "aggTime" -> SQLMetrics.createTimingMetric(sparkContext, "time in aggregation build")
+    "aggTime" -> SQLMetrics.createTimingMetric(sparkContext, "time in aggregation build"),
+    "spillSize" -> SQLMetrics.createSizeMetric(sparkContext, "spill size"),
+    "fallBackToSortBasedAgg" -> SQLMetrics.createMetric(sparkContext,
```

Comment:
The name is a bit misleading, as the fallback happens only once for each task. Probably `numTasksFallBacked`?

Reply:
@cloud-fan - sure, updated.